aiosmtpd/tests/test_smtp.py

```diff
@@ -590,6 +599,32 @@ def test_process_message_error(self):
                 self.assertEqual(cm.exception.response,
                                  b'Could not accept the message')
 
+    def test_too_long_message_body(self):
+        controller = SizedController(Sink(), size=100)
+        controller.start()
+        self.addCleanup(controller.stop)
+        with SMTP(controller.hostname, controller.port) as client:
+            client.helo('example.com')
+            mail = '\r\n'.join(['z' * 20] * 10)
+            with self.assertRaises(SMTPResponseException) as ctx:
+                client.sendmail('anne@example.com', ['bart@example.com'], mail)
+            e = ctx.exception
```

Comment:
I like to spell this out, so e.g. `error = cm.exception` (or just use `cm.exception.smtp_code` in the assertion).